packages/manager/src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerConfigurations.tsx

```diff
@@ -540,6 +545,18 @@ class NodeBalancerConfigurations extends React.Component<CombinedProps, State> {
     return true;
   };
 
+  isRestrictedUser = () => {
```

Comment:
This function name is a bit misleading. Might want to change it to something like

```suggestion
  isNodeBalancerReadOnly = () => {
```